css/amp.css

```diff
@@ -751,10 +751,60 @@ amp-story .i-amphtml-loader {
  */
 
 /**
-  * In ampfx-preset-utils.js we are installing styles once the extension is 
- * loaded. This cause the element to jump. This is a fix for that, which 
+  * In ampfx-preset-utils.js we are installing styles once the extension is
+ * loaded. This cause the element to jump. This is a fix for that, which
  * doesn't push the preset's defaults into v0.js making it larger.
  */
 [amp-fx^="fly-in"] {
   visibility: hidden;
 }
+
+/* Ad lightbox styles */
+
+i-amphtml-ad-close-header {
```

Comment:
should this CSS go to lightbox?

Reply:
No, since it's rendered on the host, which may not have access  to the lightbox extension.

Reply:
does that mean even the lightbox itself is not amp-lightbox? IMO, this should fully use `amp-lightbox` by having the parent page dynamically load the extension. That may have an initial delay, but I don't think this feature can justify increase in v0.js as it would be super rare at least for time being. When most Ads are A4A and use lightbox often, we can reevaluate.